base/shared/src/main/scala/scalaz/std/byte.scala

```diff
@@ -9,7 +9,7 @@ trait ByteInstances {
   implicit val byteEq: Eq[Byte]       = universalEq[Byte]
 
   implicit val byteOrd: Ord[Byte] = instanceOf(new OrdClass[Byte] {
-    def comp(a: Byte, b: Byte) = java.lang.Byte.compare(a, b) match {
+    def comp(a: Byte, b: Byte) = _root_.java.lang.Byte.compare(a, b) match {
```

Comment:
Any reason to add `_root_` other than avoiding potential conflicts with a future `scalaz.java` package?

Reply:
No, there's literally a bug in the incremental compiler which told me I imported from `scalaz.std.java`, despite that package never existing and despite doing a clean. :man_shrugging: